examples/cloud-support/java_update_cases_blog_code/my-app/src/main/java/com/mycompany/app/App.java

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (C) 2022 Google Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mycompany.app;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.cloudsupport.v2beta.CloudSupport;
+import com.google.api.services.cloudsupport.v2beta.model.CloudSupportCase;
+import com.google.api.services.cloudsupport.v2beta.model.ListCasesResponse;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+// starter code for cloud support API to initialize it
+public class App {
+
+  // Shared constants
+  static final String CLOUD_SUPPORT_SCOPE = "https://www.googleapis.com/auth/cloudsupport";
+
+  // TODO: update PROJECT with your project number
+  static final String PROJECT = "projects/123456789";
```

Comment:
can you make this some kind of input instead of hard coding it?

Reply:
It's definitely possible, but I think for the purposes of the blog post it's much easier to convey that they just need to update this variable instead of giving instructions on how to pass it in and all that. Also, the goal of this code is to showcase the capability of Cloud Support API, so I think having this hard coded doesn't make a big difference. I hope this is okay, but let me know!

Reply:
Ah that makes sense! Thank you for the feedback. Will push an update to resolve this :)